Documentation/DataTypes/Index.rst

```diff
@@ -642,7 +642,45 @@ rotation
 :aspect:`Example:`
    180
 
-.. _data-type-space:
+.. _data-type-siteLanguage:
+
+siteLanguage
+----
```

Comment:
Section title underlines should be at least as long as the title. By convention used in TYPO3 docs, they should be exactly as long as title, see https://docs.typo3.org/typo3cms/HowToDocument/WritingReST/HeadlinesAndSection.html#syntax